nservicebus/eventstoretransport/index.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Event Store Transport
+summary: High-level description of NServiceBus EventStore Transport.
+component: EventStoreTransport
+tags:
+ - EventStore
+ - GetEventStore
+---
+
+The Event Store transport implements a message queuing mechanism on top of [Event Store](https://geteventstore.com/) stream database.
+
+
+## How it works
+
+The Event Store transport **does not** rely on the projections functionality for routing messages. Instead it maintains a routing topology data structure inside the database in a dedicated event stream `nsb-exchanges`. The structure is a simplified version of AMQP 0.9.1 routing topology. 
+
+partial:topology
+
+When a message is addressed to a given exchange `e` it is copied to all exchanges (similar as AMQP `fanout` exchange) and to all queues bound to that exchange. In the example above a message sent to `ExchangeB` would go to `Queue1` and a message sent to `ExchangeA` would go to both `Queue1` and `Queue2`.
+
+## Persistence
+
+TODO
```

Comment:
@SzymonPobiega I think 2 sentences should be enough here, how about 
"EventStore transport can be used in combination with any supported [NServiceBus persistence](/nservicebus/persistence/)" + saying what persistence is used for? (e.g. pub sub?)